modules/installation-vsphere-machines.adoc

```diff
@@ -0,0 +1,150 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/installing-vsphere.adoc
+
+[id="installation-vsphere-machines-{context}"]
+= Creating {op-system-first} machines in vSphere
+
+Before you install a user provisioned infrastructure (UPI) cluster on vSphere,
+you need to create {op-system} machines on a single host for it to use.
+
+.Prerequisites
+
+* Obtain the Ignition config files for your cluster.
+* Have access to an HTTP server that you can access from your computer and that
+the machines that you create can access.
+
+.Procedure
+
+////
+This will work for GA.
+. Obtain the {op-system} images from the
+link:https://access.redhat.com/downloads[Product Downloads] page on the Red
+Hat customer portal.
+////
+
+. Convert the bootstrap, master, and worker Ignition config files to Base64
+encoding.
++
+For example, if you use a Linux operating system, you can use the `base64`
+command to encode the files.
++
+----
+$ base64 <path_to_installation_directory>/bootstrap.ign > <path_to_installation_directory>/bootstrap.64
+$ base64 <path_to_installation_directory>/master.ign > <path_to_installation_directory>/master.64
+$ base64 <path_to_installation_directory>/worker.ign > <path_to_installation_directory>/worker.64
+----
+
+. Upload the Base64-encoded bootstrap Ignition config file that the
```

Comment:
The user should not upload the base64-encoded bootstrap Ignition config file. The user should upload the plain-text config file.